Packaging/apple/Info.plist

```diff
@@ -34,8 +40,8 @@
 	<string>${MACOSX_BUNDLE_COPYRIGHT}</string>
 	<key>SDL_FILESYSTEM_BASE_DIR_TYPE</key>
 	<string>parent</string>
-	<key>NSPrincipalClass</key>
-	<string>NSApplication</string>
+	<key>NSBluetoothAlwaysUsageDescription</key>
+	<string>Connect with gamepads</string>
```

Comment:
NSBluetoothAlwaysUsageDescription should not be needed now that we use SDL 2.0.18+

Reply:
going to check this now